WEB_GUIDE.md

```diff
@@ -468,6 +470,148 @@ callback("Error message");
+| "$windows_phone_url" | Same as above but for Windows Store
+| "$after_click_url" | When a user returns to the browser after going to the app, take them to this URL. _iOS only; Android coming soon_
+
+You have the ability to control the direct deep linking of each link as well:
+
+| Key | Value
+| --- | ---
+| "$deeplink_path" | The value of the deep link path that you'd like us to append to your URI. For example, you could specify "$deeplink_path": "radio/station/456" and we'll open the app with the URI "yourapp://radio/station/456?link_click_id=branch-identifier". This is primarily for supporting legacy deep linking infrastructure.
+
+#### Usage
+```js
+branch.deepview(
+    data,
+    options,
+    callback (err)
+);
+```
+
+#### Example
+```js
+branch.deepview(
+    {
+        channel: 'facebook',
+        data: {
+            mydata: 'content of my data',
+            foo: 'bar',
+            '$desktop_url': 'https://en.wikipedia.org/wiki/Internet',
+            '$ios_url': 'https://en.wikipedia.org/wiki/Internet',
+            '$ipad_url': 'https://en.wikipedia.org/wiki/Internet',
+            '$android_url': 'https://en.wikipedia.org/wiki/Internet',
+            '$deepview_path': 'item_id=12345',
+            '$og_app_id': '12345',
+            '$og_title': 'My App',
+            '$og_description': 'My app\'s description.'
+        },
+        feature: 'dashboard',
+        stage: 'new user',
+        tags: [ 'tag1', 'tag2' ],
+    },
+    {
+        make_new_link: true,
+        open_app: true
+    },
+    function(err) {
+        console.log(err || 'no error');
+    }
+);
+```
+
+##### Callback Format
+```js
+callback(
+    "Error message"
+);
+```
+
+
+
+### deepviewCta(event, silent) 
+
+**Parameters**
+
+**event**: `Object`, _optional_ - from the event binding function
+
+**silent**: `boolean`, _optional_ - whether to fail silently or throw an error if branch.deepview() hasn't been called. When *event* is set, this is implied to be *true*.
+
+Perform the branch deepview CTA (call to action) on mobile. Namely, depends on how
+*branch.deepview* is set up, the mobile users are redirected accordingly. If the deepview is
+configured with the option *`open_app`* being true, an immediate attempt is made as soon as
+deepview finishes, and thus the CTA is to visit the platform-appropriate app stores; if on the
+other hand, the deepview is configured with the option *`open_app`* being false, the CTA is to
+try to open app, and to visit the platform-appropriate app stores if the open-app attempt failes.
+
+If *branch.deepview* has not been called and *silent* is not set to true, an error will arise
+with a reminder to call *branch.deepview* first.
+
+##### Usage
+```js
+// If you're using jQuery, then:
+$('a.deepview-cta').click(branch.deepviewCta);
+
+// Or you can:
+document.getElementById('my-elem').onClick = branch.deepviewCta;
+
+// Or in HTML you can:
+<a href='...' onclick='branch.deepviewCta(null, true); return false'>
```

Comment:
you don't need to return false since branch.deepviewCta already returns false. i think it's fine to remove the second param also